eng/Packages.Data.props

```diff
@@ -108,7 +108,7 @@
     <!-- Azure SDK packages -->
     <PackageReference Update="Azure.AI.Inference" Version="1.0.0-beta.2" />
     <PackageReference Update="Azure.AI.OpenAI" Version="2.0.0" />
-    <PackageReference Update="Azure.AI.Agents" Version="1.0.0-beta.1" />
+    <PackageReference Update="Azure.AI.Agents" Version="1.0.0-alpha.20250211.4" />
```

Comment:
We generally do not allow for dev feed versions here.   Please use a project reference instead of package reverence in the `.csproj` file.

Reply:
Fixed.